Swift/Tests/CollectionTest.swift

```diff
@@ -18,7 +18,8 @@
 //
 
 import XCTest
-import CouchbaseLiteSwift
+@testable import CouchbaseLiteSwift
+@testable import CouchbaseLiteSwift_Private
```

Comment:
CouchbaseLiteSwift_Private mostly contains objective-c classes. Is it required here in Swift Tests?

Reply:
yes as some tests check certain values with <CBL...> values